openapi.yml

```diff
@@ -4,66 +4,55 @@ info:
   version: v1
   description: Flow structures and statistics
 paths:
-  /api/amlight/flow_stats/flow/match/{dpid}:
+  /api/amlight/flow_stats/flow/stats:
```

Comment:
One of the requirements of EP32 stabilization was also to bump v1 on endpoint routes, could you update it accordingly since we're going for a stabilization?

Reply:
@viniarck, I have to modify the other endpoints too, right?

Reply:
@gretelliz, yes, putting a `v1` in the route before of the name of the resource like `/api/amlight/flow_stats/v1/flow/stats`.

Reply:
We'll need this `v1` on all routes that you're stabilizing and making the v1 official.